node-fs-readfile/cat.js

```diff
@@ -0,0 +1,17 @@
+const fs = require('fs');
+var files = [];
+
+function readingFiles(filePath) {
+  fs.readFile(filePath, 'utf8', (err, data) => {
+    if (err) throw err;
+    files.push(data);
```

Comment:
push won't guarantee the print order, but if you can somehow track the index and keep a count of how many files are remaining, that could work instead of just pushing to the back of the array.